README.md

```diff
@@ -1 +1,43 @@
 ## TODO: create readme
+
+### Introduction: 
+The project involves developing a POS system user interface and an ECommerce website that integrate with it. The POS system is designed to be used by cashiers or managers to manage sales, inventory. And users can ultilize the website to pick orders, see the inventory amount, and place orders which they are being processed by the POS system.
+### Features
+### Point of Sale System:
+**- User Interface:** Create a simple UI for the cashier/manager to be able to interact with the POS system.
+
+**- Item Management:** Have the ability to create, delete, modify the items in the inventory.
+
+**- Sales Reporting:** Able to view the sales report, track the items being sold overtime and manage cash flow.
+
+**- Customization:** Business owners able to customize the system however they want, logo, themes, etc...
+### eCommerce Website:
+**- Item Display:** Displays all the available items for the customers that they are able to be ordered.
+
+**- Placement order:** Types of order, online for pickup, etc...
+
+**- Order Integration:** Things that being place ordered on the website go into the POS system.
+
+**- Bonus System:** Loyalty points for frequent customers (discount, bonus items, etc...)
+
+### Extremely Stretchness Goalmericas
+**- Stripe Integration:** The feature allows the use of credit/debit cards transaction on the POS system and website
+
+### Installation
+### Have:
+- Node.js
+  
+- PostgresSQL
+
+### Initialize
+- npm install
+
+### Usuage (Both eComs and POS)
+- npm start
+- node server.js
+
+### Link name
+**- POS:** http://localhost:3000/pos
+
+**- eCom:** http://localhost:3000/
```

Comment:
I don't think we need these at the moment, since we are still deciding on the endpoints